# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/platforms/PlatformEncoderModule.h
# Commit: bcb45500ba22
# Full Hash: bcb45500ba2223a5e21bcc29922c1260d350694f
# Author: Paul Adenot <paul@paul.cx>
# Date: 2023-12-23 09:52:03
# Regressor Bug: 1749047
# File Overlap Count: 3
# Description:
#   Bug 1749047 - Wire up more codecs from VideoEncoder to the new FFmpeg PEM. r=chunmin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D196108
# ==============================================================================

diff -r a78bb8799ef4 -r bcb45500ba22 dom/media/platforms/PlatformEncoderModule.h
--- a/dom/media/platforms/PlatformEncoderModule.h	Fri Dec 22 21:39:12 2023 +0000
+++ b/dom/media/platforms/PlatformEncoderModule.h	Fri Dec 22 21:39:13 2023 +0000
@@ -32,6 +32,7 @@
   H264,
   VP8,
   VP9,
+  AV1,
   _EndVideo_,
   _BeginAudio_ = _EndVideo_,
   Opus,